exporters/trace/jaeger/uploader.go

```diff
@@ -41,9 +41,15 @@ type EndpointOption func() (batchUploader, error)
 // For example, localhost:6831.
 func WithAgentEndpoint(agentEndpoint string, options ...AgentEndpointOption) EndpointOption {
```

Comment:
I'm not sure having `agentEndpoint` as an argument here makes sense anymore given we have two other ways it can be set (env var, and then default). We should remove this arg and make into a `WithAgentHost` and `WithAgentPort`, or maybe just `WithAgentEndpoint`(?), options.

Reply:
Removed the `agentEndpoint` argument from `WithAgentEndpoint` function. Didn't add in `WithAgentHost` or `WithAgentPort`, not sure what is the benefit?

Reply:
We will need `WithAgentHost` and `WithAgentPort` to be added so the user can specify these options in code instead of requiring they use just environment variables.